### PR TITLE
[ALLUXIO-2400] Make MAX_RETRIES in TieredBlockStore configurable

### PR DESCRIPTION
**Pull request by luoli523**
https://alluxio.atlassian.net/browse/ALLUXIO-2400

In TieredBlockStore, the MAX_RETRIES is a hard code, it might be a good choice to make it configurable.

### CODE REVIEW

**Comment by apc999** [comment]
+1 LGTM. thanks for the patch!

**Comment by calvinjia** [comment]
@luoli523 Thanks for the improvement, could you put the docs in alphabetical order? Otherwise this looks good

**Comment by luoli523** [comment]
@calvinjia  I updated the patch for alphabetic order, Please check it again. Thanks.

File: core/server/src/main/java/alluxio/worker/block/TieredBlockStore.java [hunk 1]
```diff
@@ -80,8 +82,8 @@
 @NotThreadSafe // TODO(jiri): make thread-safe (c.f. ALLUXIO-1624)
 public final class TieredBlockStore implements BlockStore {
   private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
-  // TODO(bin): Change maxRetry to be configurable.
-  private static final int MAX_RETRIES = 3;
+  private static final int MAX_RETRIES =
+          Configuration.getInt(PropertyKey.WORKER_BLOCK_HEARTBEAT_TIMEOUT_MS);
```
Comment (apc999) [inline core/server/src/main/java/alluxio/worker/block/TieredBlockStore.java]: actually, this is should be `WORKER_TIERED_STORE_RETRY`

Comment (luoli523) [inline core/server/src/main/java/alluxio/worker/block/TieredBlockStore.java]: @apc999  OMG, Sorry for the uncareful, I have fix it.

**Comment by luoli523** [comment]
@apc999 Sorry for the mistake. I have fix that.

**Pull request merged**